src/plugins/hiddenRows/test/plugins/contextMenu.e2e.js

```diff
@@ -213,6 +213,26 @@ describe('HiddenRows', () => {
           expect(actions.text()).toEqual(MENU_ITEM_SHOW_ROWS);
         });
 
+        it('should render proper context menu item for unhiding if all rows are hidden', () => {
```

Comment:
Could you add analogous tests for the `HiddenColumns` plugin?

Reply:
I'll do that in a separate issue.